db/migrate/20220721122800_add_last_login_time_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddLastLoginTimeToUsers < ActiveRecord::Migration[6.0]
```

Comment:
I do not agree with this change from UI/UX perspective. Will our users be happy when we capture this information?

Reply:
This property was added to enable tracking of when the account creation request was sent. It was also meant  to be an additional security feature for users to monitor when their account has been logged into, similar to banking apps.
![image](https://user-images.githubusercontent.com/24633766/182269482-94865c3e-7060-4986-9537-2470a3670b1e.png)
Of course this feature may not be liked by more privacy oriented users! I am open to removing this feature if need be.